### PR TITLE
Improve contribution guidelines

### LINKED ISSUE #27: Improve contribution guidelines
Current contribution guidelines suggests use of `git merge` on master after fetching upstream. There is another way to update master through `git rebase`.

To solve this issue:
- [ ]  Do some self research and state differences between both commands and make the explanation apparent in commit message.
- [ ] Mention these intricacies in contribution guidelines.
- [ ] Add reasons explaining which command is preferred.

### PR DESCRIPTION
**Pull request by pranshi112**
fixes #27 

`git merge` integrates the histories of the upstream branch and your master branch. Merging preserves history since the existing branches are not changed in any way.
`git rebase` moves the entire upstream branch on the tip of your master branch. Rebasing re-writes history by creating brand new commits for each commit in the original branch.

- [x] Do some self-research and state differences between both commands and make the explanation apparent in commit message.
- [x] Mention these intricacies in contribution guidelines.
- [x] Add reasons explaining which command is preferred.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,22 @@ At this point, your local branch is synced to the original repository's master b
...
+
+#### Alternative
+
+Set up  the original repository as your upstream.
+```
+git remote add upstream https://github.com/The-Codesis/simple-static-website.git
+```
+Now, fetch and merge the changes from the original repository simultaneously.
+```
+git pull upstream master
```
Comment (adiaholic) [inline README.md:83]: @pranshi112 can you look for the difference between `git pull upstream` and `git fetch upstream`

**Comment by pranshi112** [comment]
I’ve already looked that :)

File: README.md (hunk 1 above)
Comment (pranshi112) [reply README.md:83]: What kind of changes? Please elaborate.

Comment (tushar5526) [reply README.md:83]: git fetch upstream do not merge it with your local repo, just download the changes whereas git pull upstream downloads and merges them in one go, you can add that if you want

Comment (adiaholic) [reply README.md:83]: @tushar5526 Right

Comment (pranshi112) [reply README.md:83]: @tushar5526 Right

**Pull request merged**